concepticondata/conceptlists.tsv

```diff
@@ -1,4 +1,5 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
+Sidwell-2021-100	Sidwell, Paul	2021		100	basic	English	Nicobarese		Sidwell2021		The data set is based on a modified Swadesh 100 list, intended to provide indications of the internal branching of the Nicobarese languages.		
```

Comment:
Please change the note to
```
This concept list is taken from a dataset on Nicobarese languages. The author has also indicated whether the concepts recur in [Swadesh's list of 100](:ref:Swadesh-1955-100) and [200 items](:ref:Swadesh-1952-100).
```
@mathildavz, we had this already, but I think it is interesting to remind, that we can use the `[X](:ref:CONCEPTLIST)` or the `[Y](:bib:BibtexKey)` construct to refer to lists in concepticon which are later represented as a URL in the website. 

In general, we don't talk of "Swadesh 100" but of "Swadesh's list of 100 items". As we want to have a stricter terminology, and Swadesh 100 sounds very technical, excluding people from the discourse who are no experts on Swadesh's work.